Library/src/Systems/Assets/AssetCollection.cpp

```diff
@@ -93,12 +94,15 @@ void PrototypeDtoToAssetCollection(const chs::PrototypeDto& Dto, csp::systems::A
 
 	if (Dto.HasGroupIds())
 	{
-		auto& GroupIds			 = Dto.GetGroupIds();
-		AssetCollection.SpaceIds = csp::common::Array<csp::common::String>(GroupIds.size());
-
-		for (int i = 0; i < GroupIds.size(); ++i)
+		auto& GroupIds = Dto.GetGroupIds();
+		if (GroupIds.size() > 0)
+		{
+			AssetCollection.SpaceId = GroupIds[0];
+		}
+		else
 		{
-			AssetCollection.SpaceIds[i] = GroupIds[i];
+			CSP_LOG_MSG(csp::systems::LogLevel::Error,
+						"Encountered an asset collection that did not belong to any space whilst processing a response from services.");
```

Comment:
It is valid for an asset collection to not be associated with a space, so this should not log an error

Reply:
Addressed in latest. Please can I get a re-review @MAG-AW @MAG-AdamThorn?